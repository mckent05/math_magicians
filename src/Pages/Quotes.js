import React from 'react';

const Quotes = () => (
  <div className="quote">
    <p>
      &quot;Mathematics is the queen of science
      and arithemetic the queen of mathematics&quot; - Carl Friedrich Gauss
    </p>
    <p>
      &quot;If people do not
      believe that mathematics
      is simple, it is only because
      they do not realize how complicated life is&quot; - Carl Friedrich Gauss
    </p>
  </div>
);

export default Quotes;
