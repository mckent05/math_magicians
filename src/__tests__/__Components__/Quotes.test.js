import React from 'react';
import { render, unmountComponentAtNode } from 'react-dom';
import { act } from 'react-dom/test-utils';
import pretty from 'pretty';

import Quotes from '../../Pages/Quotes';

let container = null;
beforeEach(() => {
  // setup a DOM element as a render target
  container = document.createElement('div');
  document.body.appendChild(container);
});

afterEach(() => {
  // cleanup on exiting
  unmountComponentAtNode(container);
  container.remove();
  container = null;
});

it('testing for the UI of QuotePage', () => {
  act(() => {
    render(<Quotes />, container);
  });

  expect(pretty(container.innerHTML)).toMatchSnapshot();
});
