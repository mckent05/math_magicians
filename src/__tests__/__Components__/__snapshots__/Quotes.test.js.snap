// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing for the UI of QuotePage 1`] = `
"<div class=\\"quote\\">
  <p>\\"Mathematics is the queen of science and arithemetic the queen of mathematics\\" - Carl Friedrich Gauss</p>
  <p>\\"If people do not believe that mathematics is simple, it is only because they do not realize how complicated life is\\" - Carl Friedrich Gauss</p>
</div>"
`;
